.github/workflows/ci.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: Test
         run: cargo test --release
 
+      - name: Format
+        run: cargo +nightly fmt
```

Comment:
Actually formatting the code in actions doesn't probably make sense. We instead rather want to check whether formatting is okay:

```suggestion
        run: cargo +nightly fmt -- --check
```

Reply:
Totally agree. I just forgot to use `--check` and wanted to see if `+nightly` would work at this place (discussed it with @jxs a week ago).